### DISCUSSION TITLE
Wireframe

### DISCUSSION THREAD

**Discussion by kyleofawesome** [gazzabwoy/projectEprom#15] [Show and tell]
Hello everybody I've finished the wireframe and just wanted you opinion on it before I upload it

 

![The HomePage](https://user-images.githubusercontent.com/35610708/141527703-4976c842-768a-42e6-9038-6972813ba22a.PNG)

![The Game Pages (1 2   3)](https://user-images.githubusercontent.com/35610708/141527716-e11dcf6c-2a4f-47a1-8216-f9e0b5bfc7cb.PNG)

![The Booking Page](https://user-images.githubusercontent.com/35610708/141527728-751ea93c-9f95-48a9-a737-2c8ca568576f.PNG)

![Location Page](https://user-images.githubusercontent.com/35610708/141527737-71addc7a-5c04-4d4e-b2b9-c3a6af73ae58.PNG)

**Comment by gazzabwoy** [comment]
![Index html](https://user-images.githubusercontent.com/91916706/141531617-02c35b0f-0fa5-467b-8349-6fc7e12686a9.png)
The question is how do you plan on implementing this?  Are you going to make a grid style or as I have done in the base code? But I want you guys to at least try and do some code.
